packages/liquidsoap-daemon/liquidsoap-daemon.0.1.2/opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "1.2"
+maintainer: "romain.beauxis@gmail.com"
+homepage: "https://github.com/savonet/liquidsoap-daemon"
+authors: "The Savonet Team <savonet-users@lists.sourceforge.net>"
+available: [ os != "win32" ]
+install: [
+  ["./daemonize-liquidsoap.sh"] { os = "linux" }
+  ["env" "init_type=launchd" "./daemonize-liquidsoap.sh"] { os = "darwin" }
+]
+remove: [
+  ["env" "mode=remove" "./daemonize-liquidsoap.sh"] { os = "linux" }
+  ["env" "mode=remove" "env" "init_type=launchd" "./daemonize-liquidsoap.sh"] { os = "darwin" }
+]
+depends: [
+  "liquidsoap"
```

Comment:
Thanks @yallop!  If you could push a merge commit from opam-repository:master to your branch @toots, the CI should retest with the #8602 fix included.